conda.recipe/meta.yaml

```diff
@@ -0,0 +1,105 @@
+{% set data = load_setup_py_data() %}
+
+package:
+  name: tethysplatform
+  version: {{ data['version'] }}
+
+
+source:
+  path: ..
+
+build:
+  number: 0
+  noarch: python
+#  script:  python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python >=3.6
+
+    # system dependencies
+    - pycrypto
```

Comment:
We discussed locking the versions of dependencies for the conda build. What's the most straight forward way to do that?

Reply:
We just need to add all of the version pins into the `meta.yaml` file. Alternatively we could add all of these dependencies to the `setup.py` file and then copy them from there for the conda recipe.